app/src/src/components/views/EditProfileView.jsx

```diff
@@ -4,308 +4,320 @@ import {
 
 const EditProfileView = ({ user, navigation }) => {
-  // useStates for data
-  const [bgPic, setbgPic] = useState();
-  const [pfp, setPfp] = useState();
-  //const [pfpFile, setPfpFile] = useState();
-  //const [bgPicFile, setbgPicFile] = useState();
+  // useStates for user data
+  const [bgPic, setbgPic] = useState(null);
+  const [pfp, setPfp] = useState(null);
   const [prefName, setPrefName] = useState(user.preferred_name);
+  const [fName, setFName] = useState(user.first_name);
+  const [lName, setLName] = useState(user.last_name);
+  const [email, setEmail] = useState(user.email);
   const [phone, setPhone] = useState(user.phone_number);
   const [pronoun, setPronoun] = useState(user.pronouns);
   const [birth, setBirth] = useState(user.birthday);
-  const [reg, setRegion] = useState(user.community);
-  const [program, setProgram] = useState(user.enrolled_program);
-  const [term, setTerm] = useState(user.current_term);
+  const [comm, setComm] = useState(user.community);
+  const [bio, setBio] = useState(user.bio);
 
   // useStates for modal
-  const [visiblePronoun, setVisiblePronoun] = useState(false);
-  const [visibleReg, setVisibleReg] = useState(false);
-  const [visiblePro, setVisiblePro] = useState(false);
-  const [visibleTerm, setVisibleTerm] = useState(false);
   const [visibleBirth, setVisibleBirth] = useState(false);
-  const [visiblePics, setVisiblePics] = useState(false);
 
   // useState for datetimepicker
   const [date, setDate] = useState(new Date());
 
   // useStates for stupid react native picker defaulting 0
   const [zeroPronoun, setZeroPronoun] = useState();
   const [zeroReg, setZeroReg] = useState();
-  const [zeroPro, setZeroPro] = useState();
-  const [zeroTerm, setZeroTerm] = useState();
 
   const handleEdit = async () => {
-    const result = await ProfileEndpoint.patch({
-      //profile_photo: pfp ? pfpFile : user.profile_photo,
-      //background_photo: bgPic ? bgPicFile : user.background_photo,
-      preferred_name: prefName ? prefName : user.preferred_name,
-      phone_number: phone ? phone : user.phone_number,
-      pronouns: zeroPronoun == 0 ? 0 : pronoun ? pronoun : user.pronouns,
-      birthday: birth ? birth : user.birthday,
-      community: zeroReg == 0 ? 0 : reg ? reg : user.community,
-      enrolled_program:
-        zeroPro == 0 ? 0 : program ? program : user.enrolled_program,
-      current_term: zeroTerm == 0 ? 0 : term ? term : user.current_term,
-    });
-    if (result) {
-      await setUserData(result.data);
-      navigation.navigate(routes.profile);
-    }
-  };
-
-  useEffect(() => {
-    const askUser = async () => {
-      if (Platform.OS !== 'web') {
-        const status = await ImagePicker.requestCameraRollPermissionsAsync();
-        if (status !== 'granted') {
-          alert('We need your camera roll permissions to change your pictures');
-        }
+    try {
+      const result = await ProfileEndpoint.patch({
```

Comment:
For consistency, I'd rename 'result' to 'response' since it's the full AxiosResponse object